clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts

```diff
@@ -82,67 +82,59 @@ export class CopybookDownloadService implements vscode.Disposable {
     return action === UNLOCK_DOWNLOAD_QUEUE_MSG;
   }
 
-  private static isInvalidCredentials(e: any) {
+  public isInvalidCredentials(e: any) {
     return e?.mDetails?.errorCode === 401;
   }
 
-  private static async fetchCopybook(
+  public async fetchCopybook(
     dataset: string,
     copybookProfile: CopybookProfile,
     isUSS: boolean,
   ): Promise<boolean> {
     let members: string[] = [];
     try {
-      members = await CopybookDownloadService.getAllMembers(
+      members = await this.getAllMembers(
         dataset,
         copybookProfile.profile,
         isUSS,
       );
     } catch (error: any) {
-      if (CopybookDownloadService.isInvalidCredentials(error)) {
+      if (this.isInvalidCredentials(error)) {
         throw error;
       }
       if (!copybookProfile.quiet) {
         vscode.window.showErrorMessage(error.message);
       }
       return false;
     }
-    const remoteCopybook = CopybookDownloadService.getRemoteCopybookName(
+    const remoteCopybook = this.getRemoteCopybookName(
       members,
       copybookProfile.getCopybook(),
     );
     if (!remoteCopybook) {
       return false;
     }
     copybookProfile.updateCopybook(remoteCopybook);
-    await CopybookDownloadService.downloadCopybookFromMFUsingZowe(
-      dataset,
-      copybookProfile,
-      isUSS,
-    );
+    this.downloadCopybookFromMFUsingZowe(dataset, copybookProfile, isUSS);
```

Comment:
```suggestion
    await this.downloadCopybookFromMFUsingZowe(dataset, copybookProfile, isUSS);
```

Reply:
should be resolved: 9be6c0a07652acd9800635c19569de87906bf6ce